src/main/java/com/empowerhack/family/memberservice/Repository/MemberRepository.java

```diff
@@ -2,10 +2,12 @@
 
 import java.util.List;
 import com.empowerhack.family.memberservice.Entity.Member;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.data.repository.query.Param;
 
 public interface MemberRepository extends PagingAndSortingRepository<Member, Long> {
 
+    @Query("SELECT m FROM Member m WHERE m.name LIKE %:name%")
```

Comment:
Do you not need to escape the literal in single quotes? Would that not render to %'(value)'%